frontend/src/employee-frontend/components/vasu/components/LeaveVasuPageButton.tsx

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+import styled from 'styled-components'
+import { useHistory } from 'react-router'
+import Button from 'lib-components/atoms/buttons/Button'
+import { UUID } from 'lib-common/types'
+import { useTranslation } from '../../../state/i18n'
+
+export const LeaveVasuPageButton = React.memo(function LeaveVasuPageButton({
+  childId
+}: {
+  childId: UUID
+}) {
+  const history = useHistory()
+  const { i18n } = useTranslation()
+
+  return (
+    <ExitButtonWrapper>
+      <Button
+        text={i18n.vasu.leavePage}
+        onClick={() => history.push(`/child-information/${childId}`)}
+      />
+    </ExitButtonWrapper>
+  )
+})
+
+const ExitButtonWrapper = styled.div`
+  flex: 1;
+`
+
+export const FullWidthDiv = styled.div`
+  width: 100%;
+`
```

Comment:
`FullWidthDiv` is not used by `LeaveVasuPageButton` so it should probably be defined somewhere else.

Reply:
Moved to lib-components/layouts